content/program-testing.md

```diff
@@ -0,0 +1,534 @@
+# Program Testing
+
+# Lesson Objectives
+
+*By the end of this lesson, you will be able to:*
+
+- Describe various ways to test Solana programs
+- Explain the difference between unit tests, integration tests, and RPC tests
+- Debug Solana programs
+
+# TL;DR
+
+- Testing is a **key component** of smart contract development, it ensures the code works as intended before releasing it to the public.
```

Comment:
@ixmorrow awk

sugg: 
> - Testing is a **key component** of smart contract development because it ensures the code works as intended before releasing it to the public.

Reply:
@ixmorrow just checking in - can you resolve these comments, suggestions and notes on this PR/103?